### ISSUE TITLE
Strange behaviour with crescendo and diminuendo beta 15

### ISSUE THREAD

**Issue by musikboden** [paulrosen/abcjs#504]
There is some strange playback behaviour when using crescendo and diminuendo in abcjs
I tried this test in "editor-synth" example from beta 14

X: 1
T: Crescendo & Diminuendo
M: 4/4
L: 1/8
K: C
!p!C!<(!DEF GABc |d2 B2 G2 F2!<)! | !f!E!>(!FGA Bcde!>)! | !p!f2 d2 B2 A2 |
G2 c2 e2 g2 | a2 f2 d2 B2 |cdBc ABGA | FGEF DEFG |
E2 C2 D2 B,2 | C8 |

![Skärmavbild 2020-08-12 kl  21 19 02](https://user-images.githubusercontent.com/18282700/90057990-75c7c480-dce1-11ea-898d-0d0e8a683fed.png)

The first crescendo from p => f has no effect on playback
The diminuendo in bar 3 is working, but at bar 5 ( second system ), the volume goes up (it should stay at 'p' )
At bar 6 playback is doing a diminuendo (there is no diminuendo written in the music), and at bar 7 the volume is zero until bar 9, where it goes up again.

Really strange behaviour.

I tried it in Google Chrome Version 84.0.4147.105

**Comment by paulrosen** [comment]
I can reproduce that. Thanks for the report! I fix it in the next beta.

**Comment by paulrosen** [comment]
fixed in beta.15

**Comment by musikboden** [comment]
Hi Paul, and thank you for the beta 15!
About dynamics, and in this case crescendo & diminuendo, it's still not working properly.

Some examples:
In this example there is only dynamic markings, ppp and ff, it works great, it also seperates the dynamic playback in both systems in the grand staff :)

X: 1
T: Dynamics
%%score { 1 | 2 }
M: 4/4
L: 1/4
K: C
V:1 clef=treble
!ppp!CDEF | GABc| !ff!BcBA | GFED | !ppp!C4 :|]
V:2 clef=bass
!ff!E,2 C,2 | B,,2 A,,2 | !ppp!F,,2 G,,2 | A,,2 B,,2 | C,4 :|]
![Skärmavbild 2020-08-24 kl  21 45 01](https://user-images.githubusercontent.com/18282700/91088901-13f94a00-e653-11ea-9012-99b3410ab755.png)

however if I add crescendo and diminuendo it starts to behave odd.

Here I have added crescendo and diminuendo, and I'm also using 4 p's and f's so its more easy to hear.
*The start tone is not at pppp, it's more like mf
*At the second beat in bar 2 it suddenly drops down, a tiny bit, in dynamics and then slowly get louder for the rest of the bar
*At bar 3 (ffff) it's more like mf again and at the second beat in bar 4 it suddenly gets a bit louder and slowly get more silent, and at the last note it suddenly drop down to a pppp.

I've tried without the "y" separator as well, but it makes no difference.

X: 1
T: Dynamics
%%score { 1 | 2 }
M: 4/4
L: 1/4
K: C
V:1 clef=treble
!pppp!CDEF | !<(!GABc| !<)!y!ffff!BcBA | !>(!GFED | !>)!y!pppp!C4 :|]

![Skärmavbild 2020-08-24 kl  22 06 16](https://user-images.githubusercontent.com/18282700/91090868-0b564300-e656-11ea-82c2-e150667f99a5.png)
